Neos.EventSourcedContentRepository/Tests/Behavior/Features/Bootstrap/EventSourcedTrait.php

```diff
@@ -1008,6 +842,12 @@ public function iExpectTheNodeToHaveTheProperties($nodeIdentifier, TableNode $ex
      */
     public function iExpectTheCurrentNodeToHaveTheProperties(TableNode $expectedProperties)
     {
+        // TODO hack: $this->currentNode might be stale, so we need to re-fetch it
+        $this->contentGraphInterface->resetCache();
```

Comment:
Hm, shouldn't this `resetCache` happen on the WRITE side and not inside the testcases? Does this even make sense? :)

Reply:
another runtime cache that led here. I agree, that should flush itself upon mutation